project-base/src/Shopsys/ShopBundle/Resources/views/Styleguide/styleguide.html.twig

```diff
@@ -0,0 +1,1126 @@
+                                                    </label>
+                                                </div>
+                                                <div class="box-login__item box-login__item--submit">
+                                                    <button type="submit" name="front_login_form[login]"
+                                                            class="btn id__front_login_form_login">Log in
+                                                    </button>
+                                                </div>
+                                                <div class="box-login__item box-login__item--info">
+                                                    <a href="#">Forgot your password?</a>
+                                                </div>
+                                            </div>
+                                            <input name="front_login_form[_token]"
+                                                   class="input id__front_login_form__token"
+                                                   value="mxizsxDz68IphRegvxbVu2gRxEJacy7kiuPj9Ty9UN4" type="hidden">
+                                        </form>
+                                    </div>
+                                    <a href="#" class="window-button-close window-popup__close js-window-button-close"
+                                       title="Close (Esc)"><i class="svg svg-remove-thin"></i></a>
+                                </div>
+                            </div>
+                        </div>
+                    </section>
+
+                    <section id="loaders" class="styleguide-module anchor clearfix">
+                        <h2 class="styleguide-module__title">Loaders</h2>
+                        <div style="position: relative; display: block;">
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                            labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+                            laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+                            voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
+                            cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+
+                            <div class="in-overlay__in in-overlay__in--absolute">
+                                <div class="in-overlay__spinner in-overlay__spinner--absolute">
+                                    <span class="in-overlay__spinner__icon"></span>
+                                    <span class="in-overlay__spinner__message"></span>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="styleguide-module__
```

Comment:
Is it working? I think that email link have to start at mailto:?

Reply:
I fixed it